target/linux/mpc85xx/image/Makefile

```diff
@@ -7,6 +7,17 @@ define Build/copy-file
 	cat "$(1)" > "$@"
 endef
 
+define Build/sysupgrade-dtb-tar
```

Comment:
This *still* needs to be dropped.

Reply:
Cleaned up [here](https://github.com/Hurricos/openwrt/commit/abc9d55dfd119b671aafa3777ac51a3a268988af).